### DISCUSSION TITLE
Show the zeros to the left

### DISCUSSION THREAD

**Discussion by bkameid126** [DeanIsMe/SevSeg#112] [Q&A] [answered]
In short I'm trying to do a clock using the library and I would like to know how I can display such numbers as : 

0000, 0001
00.00, 00.01

instead of 

0, 1
0.00, 0.01

thanks

**Comment by untr0py** [comment, marked as answer]
Is this what you're looking for? 

`bool leadingZeros = false; // Use 'true' if you'd like to keep the leading zeros`

**Reply by bkameid126** [reply]
yes, I couldn't find on my own. Thank you very much.
